### PR TITLE
#9606059 [ResourceDropDown] Add documentation

### PR DESCRIPTION
**Pull request by DrSammyD**

### CODE REVIEW

File: portal-sdk/generated/portalfx-controls-datetimepicker.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Be aware that [timezoneoffset](http://tantek.com/2015/218/b1/use-timezone-offset
 <a name="migrating-from-older-datetimecombo-or-datetimecombobox"></a>
 ## Migrating from older DateTimeCombo or DateTimeComboBox
 
-If you're combining two older DateTimeComboBoxes together to select a date/time range, please migrate to [DateTimeRangePicker](#portalfx-controls-datetimerangepicker.md)
+If you're combining two older DateTimeComboBoxes together to select a date/time range, please migrate to [DateTimeRangePicker](portalfx-controls-datetimerangepicker.md)
```
Comment (gurua) [inline portal-sdk/generated/portalfx-controls-datetimepicker.md]: test comment

Comment (Nadir1) [inline portal-sdk/generated/portalfx-controls-datetimepicker.md]: Why is this change here.

File: portal-sdk/templates/portalfx-create.md [hunk 2]
```diff
@@ -163,121 +163,76 @@ Learn more about [building forms](portalfx-forms.md).
...
+#### Sort
+This is a way for you to sort the values in the dropdown. It is a comparator function which takes two values from the dropdown, and returns a number for the comparison. It defaults to alphabetical based on the display string of the value.
+ 
+{"gitdown": "include-section", "file": "../../../src/SDK/Framework.Tests/TypeScript/Tests/Controls/Forms/DropDown.Subscription.test.ts", "section": "config#sort"}
+ 
+If you sort and use disable or group functionality, this will sort inside of the groups provided.
+
+### Legacy ARM dropdowns
+If you would like to replace your existing ARM dropdowns with the new accessible versions, we've created a legacy version of these dropdowns with minimal changes to the API.
+#### Subscriptions dropdown
```
Comment (rowong) [inline portal-sdk/templates/portalfx-create.md]: nit add a new line between these two sections

File: portal-sdk/templates/portalfx-create.md [hunk 3]
```diff
@@ -163,121 +163,76 @@ Learn more about [building forms](portalfx-forms.md).
...
+import * as SubscriptionDropDown from "FxObsolete/Controls/SubscriptionDropDown";
+```
+{"gitdown": "include-section", "file": "../../../src/SDK/AcceptanceTests/Extensions/SamplesExtension/Extension/Client/V1/Create/EngineV3/ViewModels/CreateEngineBladeViewModel.ts", "section": "config#subscriptionDropDown"}
+
+#### Resource groups **legacy** dropdown
+```ts
+import * as ResourceGroupDropDown from "FxObsolete/Controls/ResourceGroupDropDown";
+```
+{"gitdown": "include-section", "file": "../../../src/SDK/AcceptanceTests/Extensions/SamplesExtension/Extension/Client/V1/Create/EngineV3/ViewModels/CreateEngineBladeViewModel.ts", "section": "config#resourceGroupDropDown"}
+#### Locations **legacy** dropdown
```
Comment (rowong) [inline portal-sdk/templates/portalfx-create.md]: nit new line

File: portal-sdk/templates/portalfx-create.md [hunk 4]
```diff
@@ -163,121 +163,76 @@ Learn more about [building forms](portalfx-forms.md).
...
+```ts
+import * as LocationDropDown from "FxObsolete/Controls/LocationDropDown";
+```
+{"gitdown": "include-section", "file": "../../../src/SDK/AcceptanceTests/Extensions/SamplesExtension/Extension/Client/V1/Create/EngineV3/ViewModels/CreateEngineBladeViewModel.ts", "section": "config#locationDropDown"}
+#### Pricing dropdown
```
Comment (rowong) [inline portal-sdk/templates/portalfx-create.md]: newline

File: portal-sdk/templates/portalfx-create.md [hunk 5]
```diff
@@ -163,121 +163,76 @@ Learn more about [building forms](portalfx-forms.md).
...
-     * This extender should be the same extender view model used for the spec
-     * picker blade. You may need to extend your data context or share your data context between
-     * your create area and you spec picker area to use the extender with the current datacontext
-     */
-    specPickerExtender: new BillingSpecPickerExtender.BillingSpecPickerV3Extender(container, supplyInitialData(), dataContext)
-});
-
 ```
+{"gitdown": "include-section", "file": "../../../src/SDK/AcceptanceTests/Extensions/SamplesExtension/Extension/Client/V1/Create/EngineV3/ViewModels/CreateEngineBladeViewModel.ts", "section": "config#specDropDown"}
 #### Additional/custom validation to the ARM fields
```
Comment (rowong) [inline portal-sdk/templates/portalfx-create.md]: new line
